azure-pipelines.yml

```
@@ -45,26 +45,254 @@ variables:
+        name: CheckChanges
+        inputs:
+          targetType: 'inline'
+          script: |
+            git config --global user.email "nanoframework@outlook.com"
+            git config --global user.name "nfbot"
+
+            $auth = "basic $([System.Convert]::ToBase64String([System.Text.Encoding]::UTF8.GetBytes(":$(GitHubToken)")))"
+
+            if($env:Build_Reason -eq "Manual")
+            {
+                # this is a manual build, no need to check anything 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)